Mage.Tests/src/test/java/org/mage/test/cards/facedown/TurnFaceDownTest.java

```diff
@@ -0,0 +1,97 @@
+package org.mage.test.cards.facedown;
+
+import mage.constants.EmptyNames;
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ * @author PurpleCrowbar
+ */
+public class TurnFaceDownTest extends CardTestPlayerBase {
+
+    // MDFCs cannot be turned face down when on the battlefield
+    @Test
+    public void testTurnMDFCFaceDown() {
+        addCard(Zone.BATTLEFIELD, playerA, "Island", 5);
+        addCard(Zone.HAND, playerA, "Ixidron", 1);
+        addCard(Zone.BATTLEFIELD, playerB, "Tergrid, God of Fright", 1);
+
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Ixidron");
+
+        setStopAt(1, PhaseStep.BEGIN_COMBAT);
+        execute();
+
+        assertPermanentCount(playerB, EmptyNames.FACE_DOWN_CREATURE.toString(), 0);
+    }
+
+    // TDFCs cannot be turned face down when on the battlefield
+    @Test
+    public void testTurnTDFCFaceDown() {
+        addCard(Zone.BATTLEFIELD, playerA, "Island", 5);
+        addCard(Zone.HAND, playerA, "Ixidron", 1);
+        addCard(Zone.BATTLEFIELD, playerB, "Delver of Secrets", 1);
+
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Ixidron");
+
+        setStopAt(1, PhaseStep.BEGIN_COMBAT);
+        execute();
+
+        assertPermanentCount(playerB, EmptyNames.FACE_DOWN_CREATURE.toString(), 0);
+    }
+
+    // TODO: Currently no implemented cards can turn token creatures face down. These tests are to be uncommented after the implementation of [WHO] Cyber Conversion
```

Comment:
You can use addCustomCard with any ability and code, so no needs in paper cards for test

Reply:
Handy, but I was going to implement Cyber Conversion after this PR anyway so in the interest of not having to change these tests afterward I just went ahead and implemented Cyber Conversion in #11420